CHANGELOG.md

```diff
@@ -2,9 +2,10 @@
 
 ## [Unreleased]
 
-- Switch packaging and project workflows to poetry (internal 0.25.3)
+- Switch packaging and project workflows to poetry
```

Comment:
All other projects aligned to use past tense: 
```suggestion
- Switched packaging and project workflows to poetry
```

Reply:
I have chosen drop to stay consistent with the other entries in the changelog